charts/octavia/Chart.yaml

```diff
@@ -9,4 +9,4 @@ name: octavia
 sources:
 - https://opendev.org/openstack/octavia
 - https://opendev.org/openstack/openstack-helm
-version: 0.2.9
+version: 0.2.8
```

Comment:
we need to bump this to 0.2.9

Reply:
ok, i can see that patch merged. Let me remove the patch also